### ISSUE TITLE
UDP Servers are not receiving anything

### ISSUE THREAD

**Issue by xrv0** [xrv0/JTello#2]
Help is appreciated

**Comment by MulvadT** [comment]
Heyo did you get this to work?

**Comment by xrv0** [comment]
No, I‘ve been too busy working on other projects. Feel free to submit a PR if you added support for the Video/Status receiver.
